### ISSUE TITLE
Docs for using docker-compose for testing failing

### ISSUE THREAD

**Issue by Latrasis** [bigchaindb/bigchaindb#242]
The documentation at: http://bigchaindb.readthedocs.io/en/master/running-unit-tests.html
describes testing with the `docker-compose run bigchaindb py.test -v` command. However this just returns: 
```
$ docker-compose run bigchaindb py.test -v
Creating bigchaindb_rethinkdb-data_1
Creating bigchaindb_rethinkdb_1
usage: bigchaindb [-h] [-c CONFIG] [-y] [-v] [--experimental-start-rethinkdb]
                  {configure,show-config,export-my-pubkey,init,drop,start,load}
                  ...
bigchaindb: error: argument command: invalid choice: 'py.test' (choose from 'configure', 'show-config', 'export-my-pubkey', 'init', 'drop', 'start', 'load')
```

**Comment by diminator** [comment]
Thanks for reporting this

can you try:
```
$ docker-compose run py.test -v
```

**Comment by Latrasis** [comment]
Thanks for the quick followup! Same result it seems:
```
docker-compose run --rm bigchaindb py.test -v
Creating bigchaindb_rethinkdb-data_1
Creating bigchaindb_rethinkdb_1
usage: bigchaindb [-h] [-c CONFIG] [-y] [-v] [--experimental-start-rethinkdb]
                  {configure,show-config,export-my-pubkey,init,drop,start,load}
                  ...
bigchaindb: error: argument command: invalid choice: 'py.test' (choose from 'configure', 'show-config', 'export-my-pubkey', 'init', 'drop', 'start', 'load')

```

**Comment by r-marques** [comment]
hi @Latrasis 

The documentation in the link is old. It shouldn't even be there.
For the latest documentation check the latest tag.

We refactored our docker setup recently an currently there is no way to run tests with docker.
I will create an issue to bring that functionality back.

In the meantime you could try to run the tests without docker http://bigchaindb.readthedocs.io/en/latest/running-unit-tests.html

**Comment by ttmc** [comment]
Resolved by PR #313

The documentation was updated too:  http://bigchaindb.readthedocs.io/en/latest/running-unit-tests.html
